src/AuthApi/AuthTokenProvider.php

```diff
@@ -159,23 +174,9 @@ private function tokenRenew()
      */
     private function tokenCanBeRenewed()
     {
-        return $this->tokenExists() &&
-        time() > $this->requestTimestamp + $this->data[self::RESPONSE_KEY_REFRESH_TOKEN_TTL];
-    }
+        $refreshTokenEOL = $this->requestTimestamp + $this->data[self::RESPONSE_KEY_REFRESH_TOKEN_TTL];
```

Comment:
Pls rename to `tokenCanBeRefreshed` because when i read i thought it's copy of `tokenExpired`.